tests/e2e/nebulacluster_backup_test.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright 2023 Vesoft Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"sigs.k8s.io/e2e-framework/third_party/helm"
+
+	"github.com/vesoft-inc/nebula-operator/tests/e2e/e2ematcher"
+	"github.com/vesoft-inc/nebula-operator/tests/e2e/envfuncsext"
+
+	appsv1alpha1 "github.com/vesoft-inc/nebula-operator/apis/apps/v1alpha1"
+)
+
+const (
+	LabelCategoryBackup = "backup"
+	LabelGroupBasic     = "basic"
+)
+
+var testCasesBackup []ncTestCase
+
+func init() {
+	testCasesBackup = append(testCasesBackup, testCasesBasicBackup...)
+}
+
+// helper variables
+var enable, disable bool = true, false
+
+var storageS3 = appsv1alpha1.StorageProvider{
+	S3: &appsv1alpha1.S3StorageProvider{
+		Region:     AWSRegion,
+		Bucket:     AWSBucketName,
+		Endpoint:   AWSBucketEndpoint,
+		SecretName: AWSSecretName,
+	},
+}
+
+var storageGS = appsv1alpha1.StorageProvider{
+	GS: &appsv1alpha1.GsStorageProvider{
+		Location:   GSLocation,
+		Bucket:     GSBucketName,
+		SecretName: GSSecretName,
+	},
+}
+
+var basicBackupSpec = appsv1alpha1.BackupSpec{
+	Image:   "reg.vesoft-inc.com/cloud-dev/br-ent",
+	Version: "v3.7.0",
```

Comment:
use env variables

Reply:
Can you double check where the env variables are?

Reply:
> Can you double check where the env variables are?

@wenhaocs they're all in the tests/e2e/config/config.go file

Reply:
Correct me if I am wrong. What @MegaByte875 is saying is using env variable rather than using a fixed string like "v3.7.0" here.

Reply:
@wenhaocs Yes, the variables are declared in config.go, the actually definition happens in the environment where the test is run, such as the following lines for the e2e running in Github actions. Will also add the new variables there.
https://github.com/vesoft-inc/nebula-operator/blob/master/.github/workflows/e2e.yml#L59-L66

Reply:
Fixed